Source/RxSwift/withUnretained.swift

```diff
@@ -42,10 +42,10 @@ extension ObservableType {
      - returns: An observable sequence of tuples that contains both an unretained reference on `obj` and the values of the original sequence.
      */
     public func withUnretained<Object: AnyObject>(_ obj: Object) -> Observable<(Object, Element)> {
-        return withUnretained(obj) { ($0, $1) }
```

Comment:
Package.swift has `.v4_2` version. Is it ok to migrate to Swift 5.1 syntax? I think it's better to perform this migration in separate PR for whole repo.

Reply:
Uhm yeah, we'll just do the migration in a separate PR.

Once RxSwift 6 is out, it will only support Xcode 11 & Swift 5.1. So we'll release RxSwiftExt 6 accordingly.